k8s/nginx-http/container/nginx.conf.template

```diff
@@ -75,18 +80,21 @@ http {
       # max client request body size: avg transaction size.
       client_max_body_size 15k;
 
+      if ($request_method !~ ^(GET|OPTIONS|POST)$) {
+        return 444;
+      }
```

Comment:
Minor thing: We can move the complete block to the bottom of all the other 3 `if`s and reduce one if...else lookup in nginx.

Reply:
+1, will take are of it.